app/javascript/lib/validation/validate-file-content.js

```diff
@@ -276,99 +279,138 @@ function validateClusterCoordinates(headers) {
   return issues
 }
 
-/** Get properties about this validation run to log to Mixpanel */
-function getLogProps(fileObj, fileType, errorObj) {
-  const { file, table, delimiter } = fileObj
-  const { errors, summary } = errorObj
+/** parse a metadata file, and return an array of issues, along with file parsing info */
+export async function parseMetadataFile(chunker, mimeType) {
+  const { parsedHeaders, delimiter } = await getParsedHeaderLines(chunker, mimeType, 2)
 
-  // Avoid needless gotchas in downstream analysis
-  let friendlyDelimiter = 'tab'
-  if (delimiter === ',') {
-    friendlyDelimiter = 'comma'
-  } else if (delimiter === ' ') {
-    friendlyDelimiter = 'space'
-  }
+  let issues = validateCapFormat(parsedHeaders, delimiter)
+  issues = issues.concat(validateNoMetadataCoordinates(parsedHeaders))
+  // add other header validations here
 
-  const numRows = table.length
-  const numColumns = table[0].length
+  const dataObj = {} // object to track multi-line validation concerns
+  await chunker.iterateLines((line, lineNum, isLastLine) => {
+    const parsedLine = parseLine(line, delimiter)
+    issues = issues.concat(validateUniqueCellNamesWithinFile(parsedLine, isLastLine, dataObj))
```

Comment:
Similar to `headers` vs. `parsedHeaders`, let's refer to the lines we predominantly use in this high-level module as simply `lines` instead of `parsedLines`.

```suggestion
  await chunker.iterateLines((rawLine, lineNum, isLastLine) => {
    const line = parseLine(rawLine, delimiter)
    issues = issues.concat(validateUniqueCellNamesWithinFile(line, isLastLine, dataObj))
```